### PR TITLE
added school_level to calendar_week to be able to swap the left table…

### PR DESCRIPTION
**Pull request by GabyRangelB**
… on teacher_assignments to be calendar, and change inner join to left on join on sections

# Pull Request

## Summary & Motivation

[//]: # "When merged, this pull request will..."

## Self-review

- [x] Update **due date**, **assignee**, and **priority** on our
      [TEAMster Asana Project](https://app.asana.com/0/1205971774138578/1205971926225838)

  _If this is a same-day request, please flag that in Slack_

- [x] <kbd>Format</kbd> has been run on all modified files

- [x] Ensure you are using the `union_dataset_join_clause()` macro for queries that employ any
      models from these datasets:
  - deanslist
  - edplan
  - iready
  - pearson
  - powerschool
  - renlearn
  - titan

## Troubleshooting

- [SqlFluff Rules Reference](https://docs.sqlfluff.com/en/stable/rules.html)
- [Trunk](https://teamschools.github.io/teamster/CONTRIBUTING/#trunk)
- [dbt](https://teamschools.github.io/teamster/CONTRIBUTING/#dbt-cloud_1)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208151527790940

### CODE REVIEW

**Review by cbini — CHANGES_REQUESTED** [review]
address trunk issues

File: src/dbt/powerschool/models/intermediate/int_powerschool__calendar_week.sql
Comment (cbini) [inline src/dbt/powerschool/models/intermediate/int_powerschool__calendar_week.sql:1]: remove schools, doesn't appear to be used for anything

**Review by cbini — CHANGES_REQUESTED** [review]

File: src/dbt/powerschool/models/intermediate/int_powerschool__calendar_week.sql
Comment (GabyRangelB) [reply src/dbt/powerschool/models/intermediate/int_powerschool__calendar_week.sql:1]: meant add school levels at the end.

File: src/dbt/powerschool/models/intermediate/int_powerschool__calendar_week.sql [hunk 1]
```diff
@@ -1,4 +1,15 @@
 with
+    schools as (
+        select
+            _dbt_source_relation,
+            school_number,
+
+            case
+                high_grade when 12 then 'HS' when 8 then 'MS' when 4 then 'ES'
+            end as school_level,
+        from `kipptaf_powerschool.stg_powerschool__schools`
```
Comment (cbini) [inline src/dbt/powerschool/models/intermediate/int_powerschool__calendar_week.sql:10]: make sure you use `ref` for tables

**Review by cbini — CHANGES_REQUESTED** [review]

File: src/dbt/powerschool/models/intermediate/int_powerschool__calendar_week.sql (hunk 1 above)
Comment (GabyRangelB) [reply src/dbt/powerschool/models/intermediate/int_powerschool__calendar_week.sql]: jesus h. dhrist, what a rookie mistake. on it!

File: src/dbt/powerschool/models/intermediate/int_powerschool__calendar_week.sql [hunk 2]
```diff
@@ -1,4 +1,15 @@
 with
+    schools as (
+        select
+            _dbt_source_relation,
```
Comment (cbini) [inline src/dbt/powerschool/models/intermediate/int_powerschool__calendar_week.sql:4]: remove: it only appears on kipptaf tables

**Review by cbini — CHANGES_REQUESTED** [review]

File: src/dbt/powerschool/models/intermediate/int_powerschool__calendar_week.sql (hunk 2 above)
Comment (GabyRangelB) [reply src/dbt/powerschool/models/intermediate/int_powerschool__calendar_week.sql]: done!

**Review by cbini — APPROVED** [review]

**Pull request merged**